docs/source/getting-started/quickstart.rst

```diff
@@ -126,7 +126,7 @@ This may show multiple clusters, if you have created several:
 
   NAME       LAUNCHED     RESOURCES             COMMAND                            STATUS
   mygcp      1 day ago    1x GCP(n1-highmem-8)  sky launch -c mygcp --cloud gcp    STOPPED
-  mycluster  4 mins ago   1x AWS(p3.2xlarge)    sky exec mycluster hello_sky.yaml  UP
+  mycluster  4 mins ago   1x AWS(p4d.24xlarge)  sky exec mycluster hello_sky.yaml  UP
```

Comment:
We have since changed it to `1x AWS(p4d.24xlarge, {'A100': 8})`. May need re-alignment.